setup.py

```diff
@@ -81,17 +81,19 @@ def get_extras_require() -> Dict[str, List[str]]:
             "mpi4py",
             "mxnet",
             "nbval",
-            "pytorch-ignite",
             "scikit-image",
             "scikit-learn>=0.19.0,<0.23.0",  # optuna/visualization/param_importances.py.
-            "thop",
-            "torch==1.5.1" if sys.platform == "darwin" else "torch==1.5.1+cpu",
-            "torchvision==0.6.1" if sys.platform == "darwin" else "torchvision==0.6.1+cpu",
             "xgboost",
             "keras",
             "tensorflow>=2.0.0",
             "tensorflow-datasets",
         ]
+        + (
+            ["torch==1.6.0" if sys.platform == "darwin" else "torch==1.6.0+cpu"]
+            + ["pytorch-ignite", "thop"]
+            if (3, 5) < sys.version_info[:2]
+            else []
+        )
```

Comment:
```suggestion
        + (
            (
                ["torch==1.6.0", "torchvision==0.7.0"]
                if sys.platform == "darwin"
                else ["torch==1.6.0+cpu", "torchvision==0.7.0+cpu"]
            )
            + ["pytorch-ignite", "thop"]
            if (3, 5) < sys.version_info[:2]
            else []
        )
```

Reply:
The suggestion LGTM. I assume this addresses https://github.com/optuna/optuna/pull/1572#discussion_r464740767 but I haven't seen suggestions from PR authors before. Is this one waiting for feedback or a commit from a reviwer?